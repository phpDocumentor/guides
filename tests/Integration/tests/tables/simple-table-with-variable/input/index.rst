Some table with long lines
==========================

=========================  ==================================================
Project                    Links
=========================  ==================================================
TSconfig Reference (full)  |unknown| and another |known| variable
=========================  ==================================================

.. |known| replace:: reStructuredText
