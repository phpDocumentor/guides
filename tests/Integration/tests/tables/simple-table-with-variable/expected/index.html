<!-- content start -->
    <div class="section" id="some-table-with-long-lines">
    <h1>Some table with long lines</h1>

            <table>
                <thead>
            <tr>
                            <th>Project</th>
                            <th>Links</th>
                    </tr>
        </thead>
    <tbody>
    <tr>
            <td>TSconfig Reference (full)</td>
            <td>|unkown| and another |known| variable</td>
    </tr>
</tbody>
</table>

    </div>

<!-- content end -->
